Tweets/_posts/What is Yearn/2021-10-09-What-is-Yearn.md

```diff
@@ -4,95 +4,112 @@ title:  "What is Yearn"
 
 ![3_1446534086277861380](3_1446534086277861380.jpg)
 
-##### 4
-Yearn’s TVL has reached an all-time high of over $5bn TVL, achieving this without offering incentives. This is higher than the much-hyped Avalanche and Polygon that trade at multiples compared to YFI.
+## 4/n
+
+El TVL de Yearn ha alcanzado un máximo histórico de más de $5MM de TVL, logrando esto sin ofrecer incentivos. Esto es más que los tan publicitados Avalanche y Polygon que tienen números de negocios mucho más grandes comparados con YFI.
+
+## 5/n
 
-##### 5
-Its vaults have been integrated into Badger, Enzyme Finance, Argent Wallet, [Crypto.com](http://Crypto.com) and the MEW Wallet app on Android. Making vaults widely accessible is a great start, but Yearn is waiting for its killer dapp.
+Sus bóvedas han sido integradas en Badger, Enzyme Finance, Argent Wallet, [Crypto.com](http://Crypto.com) y la aplicación MEW Wallet en Android. Hacer que sus bóvedas sean mucho más accesibles es un gran comienzo, pero Yearn está esperando a su genial dapp.
 
 [medium.com/myetherwallet/…](https://medium.com/myetherwallet/introducing-yearn-vaults-on-mew-wallet-app-android-274818aa830e)
 
-##### 6
-The first dapp built on Yearn to get product market fit was Alchemix’s self-repaying loans. The sudden jumps in yvDai in early 2021 shown below happened as Yearn had to quickly increase the limits of its most popular vault to keep up with the demand.
+## 6/n
+
+La primera dapp desarrollada sobre Yearn para adaptar el producto al mercado fueron los préstamos autoliquidables de Alchemix. Los repentinos saltos en yvDai a comienzos de 2021 que se muestran a continuación ocurrieron cuando Yearn tuvo que aumentar rápidamente los límites de su bóveda más popular para mantenerse al día con la demanda. [pic.twitter.com/2sdXA6FrMw](https://twitter.com/Wot_Is_Goin_On/status/1446540021121617947/photo/1)
 
 ![3_1446534659123335169](3_1446534659123335169.jpg)
 
-##### 7
-[@PoolTogether\_](https://twitter.com/PoolTogether_)’s no-loss lottery also opted to earn yield with Yearn. In the UK there’s over £100bn ($140bn) in premium bonds where the lottery distributes a depressing 1% annual yield.
+## 7/n
+
+La lotería sin pérdidas de [@PoolTogether\_](https://twitter.com/PoolTogether_) también optó por obtener rendimiento con Yearn. En el Reino Unido hay más de £100MM ($140MM) en bonos premium donde la lotería distribuye un rendimiento anual deprimente del 1%.
 
-##### 8
-If [@element_fi](https://twitter.com/element_fi) can crack fixed income, with Yearn’s vaults (the “yv” tokens) in the background, then we’re talking a total addressable market in the trillions.
+## 8/n
+
+Si [@element_fi] (https://twitter.com/element_fi) puede acabar con los ingresos fijos, con las bóvedas de Yearn (los tokens "yv") en segundo plano, entonces hablamos de un mercado total direccionable de billones. [pic.twitter.com/R7mkC8h7Bq](https://twitter.com/Wot_Is_Goin_On/status/1446540026507038731/photo/1)
```

Comment:
AS-IS: "mercado total direccionable de billones".
TO-BE: "mercado total de billones".

Here I prefer to remove "direccionable" from the translation. I googled a bit that in Spanish is not common to use that word in this context, it feels strange and without it the meaning is clearer in my opinion.

Reply:
fixed